.github/workflows/test-gcp-dm.yml

```diff
@@ -14,14 +14,16 @@ on:
       - "deploy/deployment-manager/set_env.sh"
 
 env:
-  WORKING_DIR: deploy/test-environments
+  TEST_ENVS_DIR: deploy/test-environments
   INTEGRATIONS_SETUP_DIR: tests/integrations_setup
+  DEPLOYMENT_MANAGER_DIR: deploy/deployment-manager
   TF_VAR_ec_api_key: ${{ secrets.EC_API_KEY }}
   TF_VAR_ess_region: gcp-us-west2 # default region for testing deployments
 
 jobs:
-  Test-GCP-DM:
-    name: GCP Deployment Manager Test
+  # Test a GCP Deployment Manager deployment using Application Default Credentials
+  gcp_dm_adc:
+    name: CSPM GCP with ADC
```

Comment:
this job tests the code path that uses GCP's application default credentials 
https://github.com/elastic/cloudbeat/blob/e8309ef67d6bb49119a9464ff00ae0b9bf815a44/internal/resources/providers/gcplib/auth/credentials.go#L59